src/http_server/api/v1/metrics.c

```diff
@@ -271,24 +319,31 @@ void cb_metrics_prometheus(mk_request_t *request, void *data)
                 }
 
                 /* Allocate buffer */
-                sds_metric = flb_sds_create_size(k.via.str.size
+                tmp_sds =flb_sds_create_size(k.via.str.size
```

Comment:
Initialize `sds_metric`, the `tmp_sds` is just for the return value of all the `flb_sds_cat` calls. 

(I'm pretty sure this is what @edsiper intended when he asked for a temporal variable, a lot of code in FB looks like this)